landoapi/uplift.py

```diff
@@ -83,25 +132,18 @@ def create_uplift_revision(
         local_repo.approval_required is True
     ), f"No approval required for {target_repository}"
 
-    # Load release managers group for review
-    release_managers = get_release_managers(phab)
-
-    # Find the source diff on phabricator
-    stack = request_extended_revision_data(phab, [source_revision["phid"]])
-    diff = stack.diffs[source_revision["fields"]["diffPHID"]]
-
     # Get raw diff
-    raw_diff = phab.call_conduit("differential.getrawdiff", diffID=diff["id"])
+    raw_diff = phab.call_conduit("differential.getrawdiff", diffID=source_diff["id"])
     if not raw_diff:
         raise Exception("Missing raw source diff, cannot uplift revision.")
 
     # Base revision hash is available on the diff fields
-    refs = {ref["type"]: ref for ref in phab.expect(diff, "fields", "refs")}
+    refs = {ref["type"]: ref for ref in phab.expect(source_diff, "fields", "refs")}
     base_revision = refs["base"]["identifier"] if "base" in refs else None
 
     # The first commit in the attachment list is the current HEAD of stack
     # we can use the HEAD to mark the changes being created
-    commits = phab.expect(diff, "attachments", "commits", "commits")
+    commits = phab.expect(source_diff, "attachments", "commits", "commits")
     head = commits[0] if commits else None
 
     # Upload it on target repo
```

Comment:
The below code (i.e. the API call to `call_conduit`) may need to be modified so that `sourceControlSystem` is not hardcoded (currently always set to `hg` which may not always be the case). Other fields may need to be reviewed as well (e.g. `branch`). `sourcePath` may also need to be set (looks like there's a TODO there).

Lastly, `sourceControlBaseRevision` may not always point to a changeset that exists in the target repo. This may cause some problems when showing the diff on Phabricator, but also when generating the raw diff for landing. Just wanted to flag this in case it was not tested with this scenario.